src/MirrorNode.sol

```diff
@@ -51,13 +52,36 @@ abstract contract MirrorNode {
         return 0;
     }
 
+    function getAccountAddress(string memory accountId) external returns (address) {
+        if (bytes(accountId).length == 0
+            || keccak256(bytes(accountId)) == keccak256(bytes("null"))
+            || keccak256(abi.encodePacked(accountId)) == keccak256(abi.encodePacked(bytes32(0)))) {
+            return address(0);
+        }
+
+        try this.fetchAccount(accountId) returns (string memory json) {
+            if (vm.keyExistsJson(json, ".evm_address")) {
+                return vm.parseJsonAddress(json, ".evm_address");
+            }
+        } catch {
+            // Do nothing
+        }
+
+        // ignore the first 4 characters ("0.0.") to get the account number string
+        require(bytes(accountId).length > 4, "Invalid account ID, needs to be in the format '0.0.<account_number>'");
```

Comment:
Either we should perform a stricter validation or just change the message to be more precise

```suggestion
        require(bytes(accountId).length > 4, "Invalid account ID");
```